x-pack/solutions/security/plugins/security_solution/server/lib/detection_engine/rule_management/logic/detection_rules_client/detection_rules_client.ts

```diff
@@ -149,6 +151,22 @@ export const createDetectionRulesClient = ({
       });
     },
 
+    async mergeAndUpdatePrebuiltRule({
```

Comment:
Why is it required to introduce a new method? If it was the new method I'd expect it named something like `revertPrebuiltRuleCustomizations(currentRule, baseVersion)`.

Reply:
Talked with @maximpn offline about this, switching to a separate `revertPrebuiltRule` method